index.html

```diff
@@ -7,24 +7,124 @@
+          <li class="menu-item">
+            <a href="#list" class="hover:text-green-400 link-Item1">List</a>
+          </li>
+          <li class="menu-item">
+            <a href="#add_new" class="hover:text-green-400 link-Item2">Add new</a>
+          </li>
+          <li class="menu-item">
+            <a href="#contact" class="hover:text-green-400 link-Item3">Contact</a>
+          </li>
+        </ul>
+      </nav>
+      <div id="time" class="text-right mr-20 mt-4"></div>
+    </header>
     <main>
-      <article class="w-auto mt-4 text-center">
-        <section>
-          <h1 class="text-2xl uppercase font-bold text-green-400 font-serif">
-            All books are awesome
-          </h1>
-        </section>
-      </article>
-      <div id="books-container"></div>
-      <form action="#" class="w-1/6 m-auto text-right mt-8">
-        <h3 class="text-center mb-4 text-lg uppercase font-semibold text-green-400 font-serif">Add a new book</h3>
-        <input class="min-w-full border rounded border-green-400 p-1" id="book-title" type="text" placeholder="Title" /><br /><br />
-        <input class="min-w-full border rounded border-green-400 p-1" id="book-author" type="text" placeholder="Author" /><br /><br />
-        <button class="bg-green-400 rounded px-2 py-1 font-serif text-white" type="submit" id="add-book">ADD</button>
-      </form>
-    </main>
+      
+      <section class="h-screen flex flex-col items-center" id="list">
+        <h1
+          class="
+            w-auto
+            text-centertext-2xl
+            uppercase
+            font-bold
+            text-green-400
+            font-serif
+            mt-40
+            text-xl
+          "
+        >
+          All books are awesome
+        </h1>
+        <div
+          class="max-h-96 overflow-y-auto overflow-x-hidden mt-6"
+          id="books-container"
+        ></div>
+        <article class=""></article>
+      </section>
+
+      <!-- form section -->
+      <section class="h-screen flex justify-center items-center" id="add_new">
+        <form action="#" class="m-auto text-right w-1/3">
+          <h3
+            class="
+              text-center
+              mb-4
+              text-xl
+              uppercase
+              font-semibold
+              text-green-400
+              font-serif
+            "
+          >
+            Add a new book
+          </h3>
+          <input
+            class="min-w-full border rounded border-green-400 p-1"
+            id="book-title"
+            type="text"
+            placeholder="Title"
+          /><br /><br />
+          <input
+            class="min-w-full border rounded border-green-400 p-1"
+            id="book-author"
+            type="text"
+            placeholder="Author"
+          /><br /><br />
+          <button
+            class="bg-green-400 rounded px-2 py-1 font-serif text-white"
+            type="submit"
+            id="add-book"
+          >
```

Comment:
- [ ] [OPTIONAL] When I submit a book the inputs does not clear, I will suggest you add some additional functionality to clear the inputs when the user submits a book, so it is clear to the user that they have submitted the form 😄

Reply:
Okay, thanks for that.